### DISCUSSION TITLE
Install fail

### DISCUSSION THREAD

**Discussion by szirtesp** [mynaparrot/plugNmeet-server#571] [General]
Hello,

I tried to install (https://www.plugnmeet.org/docs/installation) plugNmeet server on my VPS, but it stucks at printing dots:

![image](https://github.com/user-attachments/assets/f8327a1b-28d3-4639-8fbb-806c629e62db)

In /opt/plugNmeet/log/plugNmeet.log, I found logs, generated every minutes:

{
  "file": "/go/src/app/main.go:54",
  "func": "main.startServer",
  "level": "fatal",
  "msg": "nats: no servers available for connection",
  "time": "2024-10-05T12:51:28Z"
}

What can be the problem?

**Comment by jibon57** [comment]
Try to debug to find problem of nats server.

**Reply by jibon57** [reply]
I've found one typo for nats file name. I've pushed new update. Will you please try again?

**Reply by szirtesp** [reply]
It's working. Thank you!
